### ISSUE TITLE
[App]: Sweep not showing when using Tensorflow/keras

### ISSUE THREAD

**Issue by elmondhir** [wandb/wandb#7020]
### Current Behavior

There seems to be an issue when running a sweep on a notebook, using Keras. i can see the results of each individual run but i cant find the visualization of the sweep, here is my code:

`from wandb.keras import WandbMetricsLogger, WandbModelCheckpoint
def run_sweep():
    with wandb.init():

        IMAGE_SIZE=224

        # Create the model
        model = Sequential()
        model.add(VGG16(weights="imagenet", include_top=False, input_shape=(IMAGE_SIZE, IMAGE_SIZE, 3)))
        model.add(Flatten())
        model.add(Dense(wandb.config.neurons_1, activation="relu"))
        model.add(Dense(wandb.config.neurons_2, activation="relu"))
        model.add(Dense(wandb.config.neurons_3, activation="relu"))
        model.add(Dense(4, activation="sigmoid"))

        model.layers[-6].trainable = False

        # Add WandbMetricsLogger to log metrics and WandbModelCheckpoint to log model checkpoints
        wandb_callbacks = [
            WandbMetricsLogger(),
            WandbModelCheckpoint(filepath="my_model_{epoch:02d}"),
        ]
        
        model.compile(loss='mean_squared_error',
                      optimizer='adam',
                      metrics=['accuracy'],
                      )
        #training the model
        train = model.fit(X_train, y_train, validation_data=(X_val, y_val),
                          epochs=wandb.config.epochs, 
                          callbacks = wandb_callbacks,
                          batch_size=wandb.config.batch_size, verbose=1)`
 
Then i run a sweep with my sweep config:

`sweep_config = {
    'method': 'random',
    "metric": {'goal': 'minimize', 'name': 'val_loss'},
    'parameters': {
                'epochs': {'values': [25, 50, 75]},
                'neurons_1':{'values': [128, 64, 32, 16]}, 
                'neurons_2':{'values': [128, 64, 32, 16]}, 
                'neurons_3':{'values': [128, 64, 32, 16]}, 
                'batch_size':{'values': [32, 16, 8]}, 
                }
    }
    
sweep_id = wandb.sweep(sweep_config, project="CNN-licence2")

wandb.agent(sweep_id, run_sweep, count=10)`

### Environment

OS: Windows

Browsers: Edge

Version:121

**Comment by JoanaMarieL** [comment]
Hi @elmondhir , let us take a look the details you provided and will get back to you.

**Comment by elmondhir** [comment]
Hi @JoanaMarieL i just found this solution online, by creating a custom a Custom WandbLogger class

`class WandbMetricsLoggerCustom(keras.callbacks.Callback):
    def on_epoch_end(self, epoch, logs=None):
        wandb.log(logs)`

and including it in the `callback` param instead of  
`wandb_callbacks = [
        WandbMetricsLogger(),
        WandbModelCheckpoint(filepath="my_model_{epoch:02d}"),
    ]`

This seemed to fix it. Althogh now i'm not sure if this was an app issue or a Python SDK issue.

Can you close the issue pls.
Thanks

**Comment by JoanaMarieL** [comment]
Glad to hear that @elmondhir , sure we'll close the issue and feel free to write in again for any concern. thanks!
